modules/ducktests/tests/ignitetest/services/spark.py

```diff
@@ -55,8 +55,8 @@ def __init__(self, context, num_nodes=3):
                 "collect_default": True
             }
 
-    def start(self):
-        BackgroundThreadService.start(self)
+    def start(self, clean=True):
```

Comment:
why we have clean param at the start?
AFAIU, it only suitable for restart operation?
Same question about other calls.

Reply:
when we override a method, we have to consider the arguments of the superclass